lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -605,6 +607,40 @@ class ServiceFabrikAdminController extends BaseController {
         .send({}));
   }
 
+  // Method for backup reporting
+  serviceInstanceBackupSummary(req, res) {
```

Comment:
rename method to `getServiceInstanceBackupSummary` its inline with all other method naming